test/handlers/erc721/deposit.js

```diff
@@ -1,116 +0,0 @@
-/**
```

Comment:
Revert it back. We don't use ERC721 but it is still in the repo.

Reply:
Reverted the changes